rosbag2_storage_mcap/CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ find_package(ros_environment REQUIRED)
 find_package(rosbag2_storage REQUIRED)
 find_package(ament_index_cpp REQUIRED)
 
+# Install conan
+find_package(Python3 COMPONENTS Interpreter REQUIRED)
+execute_process(COMMAND ${Python3_EXECUTABLE} -m pip install conan)
```

Comment:
Even though this command is idempotent, I would move it under the `if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")` check just below so it only runs once.